README.md

```diff
@@ -1,5 +1,6 @@
 UI
```

Comment:
Linted markdown, because it was annoying me in Visual Studio Code's error pane (I have https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)

Is it acceptable to do small changes unrelated to the main PR goal?

Reply:
No problem 👍